gridstatus/tests/test_spp.py

```diff
@@ -82,7 +82,11 @@ def test_lmp_date_range(self, market):
         Markets.REAL_TIME_5_MIN,
     )
     def test_get_lmp_historical(self, market):
-        super().test_get_lmp_historical(market=market)
+        super().test_get_lmp_historical(
+            market=market,
+            # SPP only offers interval data in for a limited period of time in the past.
+            date_str=str(pd.Timestamp.now().date() - pd.Timedelta(days=300)),
```

Comment:
The SPP interval data for 2022 is no longer available in individual files (it's one large zip for the entire year). This fixes the LMP tests by querying for more recent data.